packages/contracts-bedrock/test/Safe/LivenessModule.t.sol

```diff
@@ -310,9 +320,9 @@ contract LivenessModule_RemoveOwners_Test is LivenessModule_TestInit {
     }
 }
 
-/// @dev A copy of LivenessModule.get75PercentThreshold as a free function to use below.
-function get75PercentThreshold(uint256 _numOwners) pure returns (uint256 threshold_) {
-    threshold_ = (_numOwners * 75 + 99) / 100;
+/// @dev A copy of LivenessModule.getRequiredThreshold as a free function to use below.
+function getRequiredThreshold(uint256 _numOwners, uint256 _percentage) pure returns (uint256 threshold_) {
+    threshold_ = (_numOwners * _percentage + 99) / 100;
```

Comment:
Seems safer/more maintable to just deploy a dummy version of the module and call the public function, instead of copy/pasting the method?

Reply:
OK, that can be found here: 

https://github.com/ethereum-optimism/optimism/pull/10072#discussion_r1561490482